### PR TITLE
Updating ReadME.md

### PR DESCRIPTION
**Pull request by dswamina**
*Description of changes: Updating the Readme

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,101 +1,91 @@
...
+        let CognitoIdentityUserPoolRegion: AWSRegionType = .Unknown
+        let CognitoIdentityUserPoolId = "YOUR_USER_POOL_ID"
+        let CognitoIdentityUserPoolAppClientId = "YOUR_APP_CLIENT_ID"
+        let CognitoIdentityUserPoolAppClientSecret = "YOUR_APP_CLIENT_SECRET"
 ```
 Replace the “REPLACEME” places in each of these files with the appropriate AWS Credentials:
   *  _awsconfiguration.json_ (path: KVSiOS/Swift/KVSiOSApp/awsconfiguration.json)
-  *	 _Constants.swift_ (path: KVSiOS/Swift/KVSiOSApp/Constants.swift)
-
+  *     _Constants.swift_ (path: KVSiOS/Swift/KVSiOSApp/Constants.swift)
```
Comment (zhiyua-git) [inline README.md]: https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-ios/blob/master/Swift/KVSiOSApp/Constants.swift might provide less confusion comparing to "KVSiOS" which is not mentioned anywhere.

File: README.md [hunk 2]
```diff
@@ -1,101 +1,91 @@
...
-#####	Peer to Peer Streaming between Embedded SDK master and iOS device:
-  *	Run KVS WebRTC embedded SDK (in C) in master mode on a camera device.
-  *	Start the iOS device in viewer mode – you should be able to see the local video preview in the lower right side of the screen and also the larger part of the screen should stream the remote video view.
-
-#####	Peer to Peer Streaming between iOS device as master and Web browser as viewer:
- *	Start one iOS device in master mode for starting a new session sing a channel name (e.g. demo)
- *	Start the Web Browser using the Javascript SDK (JS with audio selected) and start it as viewer.
- *	Verify media showing up from the iOS device and also from the browser.
+#####    Peer to Peer Streaming between iOS device as master and Web browser as viewer:
+ *    Start one iOS device in master mode for starting a new session sing a channel name (e.g. demo)
```
Comment (zhiyua-git) [inline README.md]: typo? sing -> using

File: README.md [hunk 3]
```diff
@@ -1,101 +1,91 @@
...
+*    Use another iOS device to connect to the same channel name (started up in the above step set up as a master) in viewer mode. This will connect to an existing session (channel) where a master was connected previously.
 
-#### Run the iOS Sample Application
-Building the iOS sample application installs the AWSKinesisVideoWebRTCDemoApp on your iOS device. Using this app, you can verify live audio/video streaming between mobile, web and IoT device clients (camera). The procedure below describes some of these scenarios. Complete the following steps:
-1.	On your iOS device, open AWSKinesisVideoWebRTCDemoApp and login using the AWS user credentials from Set Up an AWS Account and Create an Administrator. (Note: Cognito settings can be tuned through your Cognito User Pool in the AWS management Console)
-2.	On successful sign-in, the channel configuration view is displayed where the **channel-name, client-id (optional) and region-name* have to be configured. 
+#####    Peer to Peer Streaming between Embedded SDK master and iOS device:
+  *    Run KVS WebRTC embedded SDK (in C) in master mode on a camera device.
```
Comment (zhiyua-git) [inline README.md]: Put a link https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c on KVS WebRTC embedded SDK would be good.

File: Swift/KVSiOSApp/Constants.swift [hunk 4]
```diff
@@ -1,12 +1,14 @@
 import Foundation
 import AWSCognitoIdentityProvider
 
-let cognitoIdentityUserPoolRegion = AWSRegionType.USWest2 //  <- REPLACE ME!
+// Cognito constants
+let awsCognitoUserPoolsSignInProviderKey = "UserPool"
+
+let cognitoIdentityUserPoolRegion = AWSRegionType.Unknown //  <- REPLACE ME!
```
Comment (unicornss) [inline Swift/KVSiOSApp/Constants.swift]: or we could do let cognitoIdentityUserPoolRegion = " REPLACE ME" //e.g. AWSRegionType. AWSRegionUSEast1 Ref: https://docs.aws.amazon.com/ja_jp/AWSiOSSDK/latest/Constants/AWSRegionType.html

**Review by unicornss — APPROVED** [review]

**Pull request merged**